.gitignore

```diff
@@ -9,6 +9,13 @@ __pycache__/
 build/
 dist/
 
+# Tailwind CSS
+src/argus_htmx/static/styles.css
+# Autogenerated via management command
+src/argus_htmx/tailwindtheme/tailwind.config.js
+# OS specific
+src/argus_htmx/tailwindtheme/tailwindcss
+
```

Comment:
If these are to be served with the finished package, they cannot be in .gitignore, because argus proper uses setuptools_scm to build a package. This will not include anything hit by .gitignore.

Reply:
yes, don't include `tailwindcss` in the package.

I think you do need to include the styles.css in the package. Otherwise you'd need to perform some post-installation work to generate it everywhere you install the package (such as in a deployed environment). Which *could* be done (for example,. extend the `tailwind_config` command), but you need to ensure that you have a `tailwindcss` for your platform installed.

Is there no possibility to have setuptools-scm make an exception for a specific file and include it anyway?